contrib/main.yml.sample

```diff
@@ -229,6 +229,15 @@ datadog_enabled: false
 # Only used if datadog_enabled is true.
 #datadog_api_key:
 
+# signalfx_enabled controls whether or not the SignalFX agent should be
+# deployed across the cluster. If true, signalfx_api_key must be set.
```

Comment:
To clarify this a bit, we aren't deploying the signalfx agent across the cluster. Using the addon means a single instance will be deployed via RC. How does this software actually work?